$NetBSD: patch-gtk_Makefile,v 1.1 2014/03/10 08:38:25 he Exp $

Remove references to X11 headers and libs, allow them to
be controlled by pkgsrc.  Also allow CXXFLAGS to be added to.

--- gtk/Makefile.orig	2018-06-24 18:40:43.000000000 +0000
+++ gtk/Makefile
@@ -15,12 +15,11 @@
 # along with this program; if not, see http://www.gnu.org/licenses/.
 ###############################################################################
 
-CFLAGS = -MMD \
+CFLAGS += -MMD \
 	 -Wall \
 	 -Wno-parentheses \
 	 -Wno-write-strings \
 	 -g \
-	 -I/usr/X11R6/include \
 	 $(shell pkg-config --cflags gtk+-2.0) \
 	 -DVERSION="\"$(shell cat ../VERSION)\"" \
 	 -DDECIMAL_CALL_BY_REFERENCE=1 \
@@ -29,13 +28,14 @@ CFLAGS = -MMD \
 	 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=1 \
 	 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS_ACCESS_FUNCTIONS=1
 
-CXXFLAGS = $(CFLAGS) \
+CXXFLAGS += $(CFLAGS) \
 	 -fno-exceptions \
 	 -fno-rtti \
 	 -D_WCHAR_T_DEFINED
 
 LDFLAGS = -L/usr/X11R6/lib
-LIBS = gcc111libbid.a -lXmu $(shell pkg-config --libs gtk+-2.0)
+LIBS = -lXmu $(shell pkg-config --libs gtk+-2.0)
+# XXX if bcd_math, then add gcc111libbid.a
 
 ifeq "$(shell uname -s)" "Linux"
 LDFLAGS += -Wl,--hash-style=both
